osu.Game.Rulesets.Tau/Objects/Drawables/DrawableSliderRepeat.cs

```diff
@@ -5,22 +5,43 @@
 using osu.Game.Rulesets.Tau.Objects.Drawables.Pieces;
 using osuTK;
 using System;
+using osu.Framework.Allocation;
+using osu.Game.Rulesets.Tau.UI;
 
 namespace osu.Game.Rulesets.Tau.Objects.Drawables
 {
-    public class DrawableSliderRepeat : DrawableBeat
+    public class DrawableSliderRepeat : DrawableAngledTauHitObject<SliderRepeat>
```

Comment:
Still entirely unsure about this, on one hand inheriting from `DrawableBeat` would provide us with less copy-pasting of code, but on the other hand that would require us to add `... is not SliderRepeat` again.

Reply:
I feel that would be a lot more precice in terms of the inheritence.

Reply:
I guess...